commit c53cd6e738ad627a2c5a18dad697d2accf7b5fd2
Author: Florian Weimer <fweimer@redhat.com>
Date:   Mon Jan 26 17:12:37 2026 +0100

    posix: Run tst-wordexp-reuse-mem test
    
    The test was not properly scheduled for execution with a Makefile
    dependency.
    
    Fixes commit 80cc58ea2de214f85b0a1d902a3b668ad2ecb302 ("posix: Reset
    wordexp_t fields with WRDE_REUSE (CVE-2025-15281 / BZ 33814").
    
    (cherry picked from commit bed2db02f3183e93f21d506786c5f884a1dec9e7)

diff --git a/posix/Makefile b/posix/Makefile
index 6dc240a5bf9f3046..443b1ed10bd4d564 100644
--- a/posix/Makefile
+++ b/posix/Makefile
@@ -493,7 +493,7 @@ tests-special += \
   $(objpfx)tst-pcre-mem.out \
   $(objpfx)tst-rxspencer-no-utf8-mem.out \
   $(objpfx)tst-vfork3-mem.out \
-  $(objpfx)tst-wordexp-reuse.out \
+  $(objpfx)tst-wordexp-reuse-mem.out \
   # tests-special
 endif
 endif
